honeybee_radiance_command/pinterp.py

```diff
@@ -0,0 +1,124 @@
+"""pinterp command."""
+
+from .options.pinterp import PinterpOptions
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+
+
+class Pinterp(Command):
+    """Pinterp command.
+
+    Pinterp interpolates or extrapolates a new view from one or more RADIANCE pictures 
+    and sends the result to the standard output.
+
+    Args:
+        options: Command options. It will be set to Radiance default values
+            if unspecified.
+        output: File path to the output file (Default: None).
+        view: View to interpolate or extrapolate (Default: None).
```

Comment:
Is this the view as a string? Or as a file?

Reply:
I changed this description.